packages/plugins/typescript/vue-urql/tests/__snapshots__/urql.spec.ts.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`urql Composition Should generate subscription hooks 1`] = `
+"import gql from 'graphql-tag';
```

Comment:
Is this double quote intentional? (not an authority on this, it just looks odd to me)